dart_io_mini_samples/example/http/making_multiple_requests_to_the_same_server.dart

```diff
@@ -18,16 +18,12 @@ printResponseBody(response) {
   print('...\n');
 }
 
-void main() {
-   var url = 'http://www.google.com/';
-   var client = new http.Client();
-   client.get('${url}/search')
-       .then((response) {
-         printResponseBody(response);
-         return client.get('${url}/doodles');
-        })
-       .then(printResponseBody)
-
-       // Close the connection when done.
-       .whenComplete(client.close);
+main() async {
+  var url = 'http://www.google.com/';
+  var client = new http.Client();
+  var response = await client.get('${url}/search');
+  printResponseBody(response);
+  response = await client.get('${url}/doodles');
+  printResponseBody(response);
+  client.close();
```

Comment:
to be exactly the same, shouldn't you put a try-catch around the code, and then put client.close() after that?